cmd/kube-scheduler/app/server_test.go

```diff
@@ -47,6 +51,23 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/framework"
 )
 
+func TestStatuszEndpoint(t *testing.T) {
+	testing2.SetFeatureGateDuringTest(t, feature.DefaultFeatureGate, "AllAlpha", true)
+	client := k8sclientset.New(nil)
+	sharedK8sInformerFactory := informers.NewSharedInformerFactory(client, 0)
+
+	handler := newEndpointsHandler(&config.KubeSchedulerConfiguration{}, sharedK8sInformerFactory, func() bool { return true }, nil, nil, nil)
+	req := httptest.NewRequest(http.MethodGet, "/statusz", nil)
+	req.Header.Set("Accept", "text/plain")
+	res := httptest.NewRecorder()
+	handler.ServeHTTP(res, req)
+
+	resBody := res.Body.String()
+	if !strings.Contains(resBody, "Paths") || !strings.Contains(resBody, "/configz /healthz /livez /metrics /readyz") {
```

Comment:
Wondering that the current test confirms that the expected paths are present, but it won't fail if new, unexpected paths are also returned. For example, if a /tracez endpoint were added in the future, this test would still pass.

Can we do something like whats suggested [here](https://github.com/kubernetes/kubernetes/pull/133218/files#r2265582012) for the comparison?